BeagleBone_black/src/sensors/fake_imu.cpp

```diff
@@ -38,22 +38,35 @@ using std::chrono::duration_cast;
 namespace hyped {
 namespace sensors {
 
-FakeImu::FakeImu(std::string acc_file_path, std::string gyr_file_path)
-    : pt_acc(0), pt_gyr(0)
+FakeImu::FakeImu(utils::Logger& log, std::string acc_file_path, std::string gyr_file_path)
+    : log_(log),
+      data_(data::Data::getInstance()),
+      acc_val(0),
+      gyr_val(0),
+      acc_noise(1),
+      gyr_noise(1),
+      pt_acc(0),
+      acc_file_path_(acc_file_path),
+      gyr_file_path_(gyr_file_path)
+
 {
   read_file = true;
-  readDataFromFile(acc_file_path, gyr_file_path);
+  readDataFromFile(acc_file_path_, gyr_file_path_);
   setData();
 }
 
-FakeImu::FakeImu(NavigationVector acc_val, NavigationVector acc_noise,
-                 NavigationVector gyr_val, NavigationVector gyr_noise)
-    : acc_val(acc_val), gyr_val(gyr_val),
-      acc_noise(acc_noise), gyr_noise(gyr_noise)
-{
-  read_file = false;
-  setData();
-}
+// FakeImu::FakeImu(utils::Logger& log, NavigationVector acc_val, NavigationVector acc_noise,
```

Comment:
It you are not using the code, why not delete it completely? Old code you be replaced not commented out when making updates to `develop`